docs/installation-guide/src/main/asciidoc/installing.adoc

```diff
@@ -660,81 +565,75 @@ Table 1-2 {productName} Full Profile and Web Profile Features
 |===
 |Feature |Full Platform |Web Profile
 
-|Java Platform, Enterprise Edition 8 |X |X
+|Jakarta EE ${jakartaee} Platform |X |X
 
-|Jakarta EE Security API |X |X
+|Jakarta Security ${jakarta-security-api-version} supporting OpenID Connect |X |X
```

Comment:
Using `$` causes this to be resolved as:
<img width="404" height="39" alt="image" src="https://github.com/user-attachments/assets/1291fcb0-fb34-40c0-83d8-0bfcaf1ccc87" />
so - don't.